src/main/java/com/keyboardbuilder/enterprise/services/BuildServiceStub.java

```diff
@@ -14,51 +15,34 @@
     private IBuildDAO buildDAO;
 
     @Autowired
-    private IKeyboardBuidDAO keyboardBuildDAO;
-
-
-    public BuildService() {
-
-    }
-
-    public BuildService(IBuildDAO buildDAO) {
-
-        this.buildDAO = buildDAO;
-    }
```

Comment:
Removed unnecessary constructors since spring handles this

Reply:
Good catch... You can use constructor injection with Spring, but without annotations, this does not appear to be used.  the build DAO should be decorated with the @Autowired annotation.